### ISSUE TITLE
github is deprecating the ubuntu 18.04 images

### ISSUE THREAD

**Issue by chrisbra** [vim/vim-appimage#42]
See here: https://github.com/actions/runner-images/issues/6002

we need to migrate to Ubuntu 20.04. But this may mean, that the built appimages may no longer run on older systems, since this likely goes with a newer glibc version. 

I am not sure if it is okay to drop support for older platforms or not.

**Comment by ychin** [comment]
I'm not super familiar with AppImage, but aren't the scripts we are using to build AppImage builds done in a Docker container on old CentOS versions (https://github.com/AppImage/AppImageKit#building)? If you look at the releases at https://github.com/vim/vim-appimage/releases, the glibc version is older than what's on Ubuntu 18.04. (Random note: Interestingly it went from 2.15 to 2.14 in the recent release, not sure why.

**Comment by chrisbra** [comment]
Yeah, I also noticed this glibc change. I have no idea why this dropped. 

> but aren't the scripts we are using to build AppImage builds done in a Docker container on old CentOS versions (https://github.com/AppImage/AppImageKit#building)? 

Yes, but this does not help, since the dependency is added at build time. So if we upgrade the runner to ubuntu 20.04 it might happen that the appimage requires a newer os to run on (e.g. I could imagine this be a problem on CentOS 7). I vaguely remember that this happened for Neovim as well and they are now using a Docker Container of Ubuntu 18.04 to build the artifact

**Comment by berfr3** [comment]
It's time to make a switch, @chrisbra!
